[{"name": "test.test_suicide_weather.SuicideWeather.test_calls", "status": "failed", "message": "Regex didn't match: 'Suicide DataFrame has \\\\d+ rows' not found in 'suicides_no\\nCountry                                        \\nUruguay                                0.000200\\nUzbekistan                             0.000194\\nVenezuela (Bolivarian Republic of)     0.000135\\nVirgin Islands (USA)                   0.000108\\nZimbabwe                                    NaN\\n-0.14753133069199426\\n-0.14753133069199426\\n                                    suicides_no\\nCountry                                        \\nUruguay                                0.000200\\nUzbekistan                             0.000194\\nVenezuela (Bolivarian Republic of)     0.000135\\nVirgin Islands (USA)                   0.000108\\nZimbabwe                                    NaN\\n-0.14753133069199426\\n-0.14753133069199426\\nSuicide DataFrame has  141  rows\\nTemperature DataFrame has  191  rows\\nCommon DataFrame has  108  rows\\nSpearman correlation:  -0.14753133069199426' : Output line about Suicide was incorrect!", "passed": false, "points": ["p05-07.1"], "backtrace": ["  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 615, in run\n    testMethod()\n", "  File \"/home/vins/TMC/week2/hy-data-analysis-with-python-summer-2019/part05-e07_suicide_weather/test/test_suicide_weather.py\", line 59, in test_calls\n    msg=\"Output line about Suicide was incorrect!\")\n", "  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 1316, in assertRegex\n    raise self.failureException(msg)\n"]}, {"name": "test.test_suicide_weather.SuicideWeather.test_return_value", "status": "failed", "message": "-0.14753133069199426 != -0.5580402318136322 within 4 places (0.4105089011216379 difference) : Incorrect Spearman correlation!", "passed": false, "points": ["p05-07.1"], "backtrace": ["  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 615, in run\n    testMethod()\n", "  File \"/home/vins/TMC/week2/hy-data-analysis-with-python-summer-2019/part05-e07_suicide_weather/test/test_suicide_weather.py\", line 34, in test_return_value\n    msg=\"Incorrect Spearman correlation!\")\n", "  File \"/home/vins/anaconda3/lib/python3.7/unittest/case.py\", line 893, in assertAlmostEqual\n    raise self.failureException(msg)\n"]}]